NineChronicles.Headless.Executable/appsettings.json

```diff
@@ -84,5 +88,38 @@
                 }
             }
         ]
+    },
+    "Headless": {
+        "AppProtocolVersionString": "100291/6ec8E598962F1f475504F82fD5bF3410eAE58B9B/MEUCIQCA3lzUAt0QBfG.+ezw4CQ69zBy669sANEt5juSJgzqcgIgbozfpcyeuKJDeJoT5exyGYDYBqCpxklsMEfs0SQ6qzo=/ZHUxNjpXaW5kb3dzQmluYXJ5VXJsdTcyOmh0dHBzOi8vcmVsZWFzZS5uaW5lLWNocm9uaWNsZXMuY29tL21haW4vdjEwMDI5MS9sYXVuY2hlci92MS9XaW5kb3dzLnppcHU5OnRpbWVzdGFtcHUxMDoyMDIyLTA5LTA3ZQ==",
+        "GenesisBlockPath": "https://release.nine-chronicles.com/genesis-block-9c-main",
+        "StoreType": "rocksdb",
+        "StorePath": "~/AppData/Local/planetarium/9c-main-partition",
```

Comment:
Is this path the default path used by 9c-launcher? (because 9c officially supports only Windows platform now)

Reply:
Something I'm slightly worried is that `~` is a POSIX convention and may not work on .NET or Windows.

Reply:
> Is this path the default path used by 9c-launcher? (because 9c officially supports only Windows platform now)

The launcher uses `[HOME]/AppData/Local/~~~` and finds `HOME` using [this code](https://github.com/planetarium/9c-launcher/blob/main/src/config.ts#L161-L166)
I think just put placeholder like `[path/to/9c-main-partition]` is kind of feasible but cannot sure :thinking:

Reply:
Hmm, then how about make `appsettings.json`'s `StorePath` make nullable and implement the default vlaue in the `Configuration.StorePath` property 🤔 .

Reply:
@moreal  That makes sense. :+1: